README.md

```diff
@@ -0,0 +1,34 @@
+# A guide on how to use Try Enarx 🎯
+
+A little amount of efforts are required but we will walk you through all of it. For instance, trying enarx with your own WASM files is a matter of few clicks. Our goal is to understand how it happens step by step by *learning and doing*.
+
+# Few insights on Try Enarx
+* Enarx supports [Webassembly](https://webassembly.org/) by using wasmtime as a runtime. It heavily works on it. So try to find out what these terms actually mean to us and why we use it?
+* Converting your files into WASM file can be easily done with the help of [Webassembly-guide](https://enarx.dev/docs/WebAssembly/Introduction). Also,with the language of choice.
+* Choose a language which implements Webassembly, then we're good to go.
+
+# Steps involed are: (also mentioned in a video)
+
+#### Step 1:  
+Go to [try.enarx.dev](https://try.enarx.dev/)
+
+#### Step 2:  
+Click on "Start Now" on the Try enarx page and then choose a platform: Intel/AMD on Equinix or Azure .
```

Comment:
Click on "Start Now" on the Try enarx page and then choose a platform: Intel/AMD on Equinix or Azure.